model_analyzer/config/run/run_config_generator.py

```diff
@@ -87,8 +87,13 @@ def generate_run_config_for_model_sweep(self, model, model_sweep):
         num_retries = analyzer_config['client_max_retries']
 
         if analyzer_config['triton_launch_mode'] == 'remote':
+            reload_model  = not analyzer_config['reload_model_disable']
```

Comment:
Looks like there is an extra space after `reload_model`.  Did you run formatter?

Reply:
I'm pretty sure it is set up to run the formatter when you commit, but maybe that is something I configured locally?

Reply:
Looks like the pre-commit hook it has to manually be enabled per person. Chris, see "Code Formatting" in the onboarding doc.

Reply:
Yes, running the formatter automatically before commit needs local configuration which I think you did.

Reply:
Done